### ISSUE TITLE
Search by id seems to be not working

### ISSUE THREAD

**Issue by garpinc** [Captain-P-Goldfish/scim-for-keycloak#98]
I can query a user via the endpoint via filter=username eq "simon" and it returns
```
{
  "schemas" : [ "urn:ietf:params:scim:api:messages:2.0:ListResponse" ],
  "totalResults" : 1,
  "itemsPerPage" : 1,
  "startIndex" : 1,
  "Resources" : [ {
    "schemas" : [ "urn:ietf:params:scim:schemas:core:2.0:User" ],
    "id" : "2d7e1339-1e5f-4c90-8758-e3decb737857",
    "meta" : {
      "resourceType" : "User",
      "created" : "2024-02-01T19:17:42.947Z",
      "lastModified" : "2024-02-01T19:17:42.947Z",
      "location" : "http://localhost:8080/keycloak/auth/realms/anzo/scim/v2/Users/2d7e1339-1e5f-4c90-8758-e3decb737857"
    },
    "userName" : "simon",
```

but if I do filter=id eq "2d7e1339-1e5f-4c90-8758-e3decb737857" it returns nothing

**Comment by Captain-P-Goldfish** [comment]
fixed.

**Comment by garpinc** [comment]
in 21 oracle version?

**Comment by Captain-P-Goldfish** [comment]
I haven't released the fix yet. It will be present in the next release

**Comment by garpinc** [comment]
Ok.. No rush cause i changed my code to do /Users/{id}

**Comment by Captain-P-Goldfish** [comment]
I deleted the oracle-release and instead added the release `kc-21-1.2.2-RC1-free`. Please test if your problems are resolved with this. This release contains also the fix for #97
